backend/src/core/models/steam.apps.aggregate.spec.js

```diff
@@ -125,6 +130,88 @@ describe("SteamAppsAggregate", function () {
 
       it("two games are returned", function () {
         expect(this.result.length).toBe(2);
+        expect(this.result[0]).toBeInstanceOf(Game);
+        expect(this.result[1]).toBeInstanceOf(Game);
+      });
+    });
+  });
+
+  describe(".identifyTypesViaSteamApi", function () {
+    describe("When we try to identify four steam apps", function () {
+      beforeAll(function () {
+        this.steamAppsArray = new SteamAppsAggregate(getXSampleSteamApps(4));
+
+        const steamApiApps = getXSampleSteamApiApps(3);
+
+        this.steamAppsArray.identifyTypesViaSteamApi(steamApiApps);
+      });
+
+      it("the first app is correctly identified", function () {
+        expect(this.steamAppsArray.content[0].appid).toBe(1);
```

Comment:
create expectedResult array and do the comparison with toEqual only as suggested here: https://github.com/lukatarman/steam-game-stats/pull/222/files#diff-8039836d824da0ce35a9117c5dd9973b64887008f5fcc8ff54ffc31b96106f3aR217